### DISCUSSION TITLE
Page not found (404)

### DISCUSSION THREAD

**Discussion by okcean** [WeblateOrg/weblate#9596] [Q&A]
When I visit one of the components, the page gives a 404 error. All other components are normal, would like to ask how to deal with it, thanks!
<img width="1046" alt="image" src="https://github.com/WeblateOrg/weblate/assets/50287620/499aaef7-527d-4da3-abd2-7e65ee924b18">

**Comment by nijel** [comment]
You don't have permissions to access the component.
